index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Adding the lang attribute here would be a good idea, especially for accessibility reasons.